pyiron/atomistics/job/sqs.py

```diff
@@ -94,14 +94,13 @@ def __init__(self, project, job_name):
         self._fail_early_if_imports_missing()
         self.__hdf_version__ = "0.2.0"
 
-    @staticmethod
-    def _fail_early_if_imports_missing():
+    def _fail_early_if_imports_missing(self):
         """
         Just a temporary measure as long as the imports are wrapped in a try/pass instead of being on the dependencies
         list.
         """
         try:
-            ParallelSqsIterator()
+            from sqsgenerator.core.sqs import ParallelSqsIterator
```

Comment:
Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants. https://www.python.org/dev/peps/pep-0008/#imports

Reply:
I think functionality trumps PEP8. I would rather import somewhere weird in order to fail early (i.e. when the job is instantiated) than wait until I try to run it.

Feel free to import another class from the same module at the top and just instantiate that class, but the actual class we import looks expensive to instantiate to me.

Reply:
You could also set `ParallelSqsIterator` to None in the except of the import and then check if it is None.

Reply:
Nice! I like it. Done.